.github/workflows/nightly-emscripten.yml

```diff
@@ -0,0 +1,120 @@
+name: Emscripten nightly build + solc-bin push
+
+on:
+  schedule:
+    # Run once a day, at midnight
+    - cron: '0 0 * * *'
+
+jobs:
+  build-emscripten:
+    runs-on: ubuntu-latest
+    outputs:
+      nightly-version: ${{ env.NIGHTLY_VERSION }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'ethereum/solidity'
+          ref: 'develop'
+
+      - name: Build soljson.js
+        run: |
+          # Note that this script will spawn and build inside a docker image (which works just fine in github actions).
+          scripts/build_emscripten.sh
+
+      - name: Get nightly version
+        id: nightly-version
+        run: |
+          last_commit_timestamp=$(git log -1 --date=iso --format=%ad HEAD)
+          last_commit_date=$(date --date="$last_commit_timestamp" --utc +%Y.%-m.%-d)
+          last_commit_hash=$(git rev-parse --short=8 HEAD)
+          solidity_version=$(scripts/get_version.sh)
+          nightly_version="v${solidity_version}-nightly.${last_commit_date}+commit.${last_commit_hash}"
+
+          echo "::set-env name=NIGHTLY_VERSION::${nightly_version}"
+
+      - name: Upload soljson.js as an artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: soljson.js
+          path: upload/soljson.js
+
+  commit-soljson:
+    runs-on: ubuntu-latest
+    needs: build-emscripten
+
+    env:
+      COMMITTER_NAME: emscripten nightly action
+      # TODO: Set a proper e-mail address
+      COMMITTER_EMAIL: solidity@example.com
```

Comment:
We could use solidity@ethereum.org or builds@ethereum.org

Reply:
(the latter is an account already used for the ppa builds)

Reply:
Done. Replaced with builds@ethereum.org.